packages/addons/paraglide/index.ts

```diff
@@ -17,24 +17,19 @@ import { addToDemoPage } from '../common.ts';
 const DEFAULT_INLANG_PROJECT = {
 	$schema: 'https://inlang.com/schema/project-settings',
 	modules: [
-		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern@1/dist/index.js',
-		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-identical-pattern@1/dist/index.js',
-		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation@1/dist/index.js',
-		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source@1/dist/index.js',
-		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-valid-js-identifier@1/dist/index.js',
-		'https://cdn.jsdelivr.net/npm/@inlang/plugin-message-format@2/dist/index.js',
-		'https://cdn.jsdelivr.net/npm/@inlang/plugin-m-function-matcher@0/dist/index.js'
+		'https://cdn.jsdelivr.net/npm/@inlang/plugin-message-format@4/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/plugin-m-function-matcher@2/dist/index.js'
 	],
 	'plugin.inlang.messageFormat': {
-		pathPattern: './messages/{languageTag}.json'
+		pathPattern: './messages/{locale}.json'
 	}
 };
 
 const options = defineAddonOptions({
 	availableLanguageTags: {
 		question: `Which languages would you like to support? ${colors.gray('(e.g. en,de-ch)')}`,
 		type: 'string',
-		default: 'en',
+		default: 'en, de',
```

Comment:
added a second language to have a better demo. i skipped through "which languages do you want to support?" several times. the demo will only have one language as a result leading to a "okay and now?" moment

Reply:
Agree, this makes sense. Second language was removed in #130 for whatever reason. Maybe we should use `es` instead, as spanish is one of the most spoken languages.

Reply:
```suggestion
		default: 'en, es',
```